### PR TITLE
PMM-7 add a var naming rule for compound nouns

### PR DESCRIPTION
**Pull request by atymchuk**
PMM-7

We are breaking apart what used to be "common libs" to two separate files:
1/ "tech_stack" and
2/ "best practices"

### CODE REVIEW

File: docs/process/tech_stack.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  - Use [camelCase](https://en.wikipedia.org/wiki/Camel_case)
+  - Acronyms should be all capitals, as in `ServeHTTP`
+  - Single letter represents index: `i`, `j`, `k`
+  - Short but descriptive names: `cust` not `customer`
+  - Repeat letters to represent collection, slice, or array and use a single letter in loop (`var tt []*Thing`).
+  - Avoid repeating the package name in the method name:
+    - `log.Info() // good`
+    - `log.LogInfo() // bad`
+  - Don’t name methods like `getters` or `setters`
+  - Follow the language rules when using compound nouns: `ApplicationServer` not `ApplicationsServer`. Exceptions to this rule may apply when the code is auto-generated.
```
Comment (atymchuk) [inline docs/process/tech_stack.md:82]: FYI: this was added.

File: docs/process/tech_stack.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+- [prometheus client](https://github.com/prometheus/client_golang) is used for exposing internal metrics of application and gRPC library.
+- [testify](https://github.com/stretchr/testify) or stdlib `testing` package should be used for writing tests. Testify should be used only for `assert` and `require` packages – suites here have some problems with logging and parallel tests. Common setups and teardowns should be implemented with `testing` [subtests](https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks).
+- [golangci-lint](https://github.com/golangci/golangci-lint) is used for static code checks.
+- [gocov.io](http://gocov.io/) and [gocoverutil](https://github.com/AlekSi/gocoverutil) gather code coverage metrics.
+- [Docker Compose](https://docs.docker.com/compose/) is used for a local development environment and in CI.
+- [Kong](https://github.com/alecthomas/kong) for pmm CLI and [kingpin.v2](http://gopkg.in/alecthomas/kingpin.v2) for exporters and some other code. Use [Kong](https://github.com/alecthomas/kong) if you need some new CLI or need big changes in old `kingpin.v2` CLI.
+- go modules for vendoring.
+
+## Best practices
```
Comment (BupycHuk) [inline docs/process/tech_stack.md:25]: can we split this file into 2 separate files?

Comment (atymchuk) [reply docs/process/tech_stack.md:25]: You mean, create a separate file for `Best practices`?

Comment (atymchuk) [reply docs/process/tech_stack.md]: Done!

**Review by BupycHuk — APPROVED** [review]

File: docs/process/best_practices.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Best practices
```
Comment (artemgavrilov) [inline docs/process/best_practices.md:1]: IMHO whole this page can be named `code style`

**Review by artemgavrilov — APPROVED** [review]

**Pull request merged**